Felienne/hedy
# [BUG FIX] Auto-select the first profile picture when creating a public profile

**Description**
In this PR we implement a new feature that automatically fixes a bug. We auto-select the first profile picture if no public profile is created yet. This way the user will always have a selected image and no additional error message implementation is required.

**Fixes**
This PR fixes #2218

**How to test**
Navigate to your profile and make sure you don't have a public profile yet (or remove the current one). Verify that the first image is selected in this case and you are unable to de-select a profile image, only change it.
